src/legacy/core_plugins/interpreter/__tests__/helpers/function_wrapper.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { mapValues } from 'lodash';
+
+// It takes a function spec and passes in default args into the spec fn
+export const functionWrapper = (fnSpec, mockReduxStore) => {
+  const spec = fnSpec();
+  const defaultArgs = mapValues(spec.args, argSpec => {
+    return argSpec.default;
+  });
+
+  return (context, args, handlers) =>
+    spec.fn(context, { ...defaultArgs, ...args }, handlers, mockReduxStore);
+};
```

Comment:
Just FYI I think I already copied a (slightly modified) version of this to interpreter awhile back: `src/legacy/core_plugins/interpreter/test_helpers.js`